src/decorators/tooltip/tooltip.js

```diff
@@ -1,7 +1,7 @@
 /*jshint unused:false */
 /*jshint -W025 */
 
-function(node, options) {
+function tooltip(node, options) {
```

Comment:
This IS an anonymous function that will be assigned during build, I'm not sure giving it a name is valid (see how gulp task works). It is possible there is a better way to set this up but giving it a name is solving the wrong problem.